GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -416,38 +420,6 @@ string GetDescribeInfoForRevision()
             }
         }
 
-        public void SetAvatarPosition(bool right)
```

Comment:
Looks like the avatar now always appears on the LHS, even if commit info is docked right? Can we see a screenshot of that too please? I expect it's fine.

Reply:
Ah, yes. I was meant to check it and forgot. Thank you for the reminder.

Reply:
Screenshots updated. 
There are two issues I identified:
* scrollbars are not shown, when required - I'll get it fixed in this PR
* layout autoscaling for both the left/right positions isn't working flawlessly - this will have to be dealt separately